doc_requirements.txt

```diff
@@ -1,3 +1,3 @@
 psyneulink-sphinx-theme<1.2.8.0
```

Comment:
Should the min version of `psyneulink-sphinx-theme` be set as well?

Reply:
Sphinx 5 is ok with the previous theme v1.2.5, sphinx 6 needs v1.2.6, and sphinx 7 needs v1.2.7. I wasn't really sure how best to express that here.

Reply:
yeah, sounds like the theme package is a better place to express it in its dependencies.